src/tools/rest-backend/config.cpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * @file
+ *
+ * @brief configuration helper functions
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include <iostream>
+#include <string>
+
+#include <config.hpp>
+
+namespace kdbrest
+{
+
+/* set dummy variables for statics */
+
+// JWT settings
+std::string Config::jwt_encryption_key = std::string ();
```

Comment:
Why n globals? Config shouldn't be initialized on static access? Or am I wrong and I should use a private constructor to prevent that?